client/my-sites/plans/plan-overview/plan-remove/index.jsx

```diff
@@ -0,0 +1,105 @@
+/**
+ * External dependencies
+ */
+import page from 'page';
+import React from 'react';
+
+/**
+ * Internal dependencies
+ */
+import CompactCard from 'components/card/compact';
+import Dialog from 'components/dialog';
+import { isInGracePeriod } from 'lib/plans';
+import paths from '../../paths';
+
+const PlanRemove = React.createClass( {
+	propTypes: {
+		plan: React.PropTypes.object.isRequired,
+		selectedSite: React.PropTypes.oneOfType( [
+			React.PropTypes.object,
+			React.PropTypes.bool
+		] ).isRequired
+	},
+
+	getInitialState() {
+		return {
+			showDialog: false
+		};
+	},
+
+	closeDialog( action ) {
+		this.setState( { showDialog: false } );
+
+		if ( action === 'remove' ) {
+			// TODO: Add action that calls the API and cancels the free trial
```

Comment:
I didn't know about that - I think I've followed what was the [former usage](https://github.com/Automattic/wp-calypso/blob/master/client/components/dialog/README.md) of the `<Dialog />` component. I'll update that code as you suggested.